.github/workflows/convertKqlFuncYamlToArmTemplate.yaml

```diff
@@ -0,0 +1,62 @@
+name: Convert Kql function yaml to ARM template
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize, closed, labeled]
```

Comment:
Didn't you mention that you can't run this on a closed trigger type, which is when the branch is merged?